### PR TITLE
[Feature] Prettier installation and setup

### PR DESCRIPTION
**Pull request by Orrison**
Sets up Prettier for CSS, JS, and Blade linting.

It may be best to filter out b55227c8331c78aa6eeacf440d35c3d53c9a03a1 to see the changes that are not related to me running it across the codebase.

Best to merge in after full team approval to mark agreement on the configuration.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+resources/views/vendor/mail/**/*.blade.php
+resources/views/mail/**/*.blade.php
```
Comment (dgoetzit) [inline .prettierignore:2]: Are we doing this preventatively in the event that we create markdown mail?

**Review by dgoetzit — APPROVED** [review]
Looks good! 1 non-blocking question.

File: .prettierignore (hunk 1 above)
Comment (Orrison) [reply .prettierignore:2]: Yeah, exactly, it causes a lot of issues before so just getting out ahead of it just in case it is ever added.

Comment (joelicatajr) [reply .prettierignore:2]: This makes sense.

**Review by joelicatajr — APPROVED** [review]

**Pull request merged**